external/header.php

```diff
@@ -67,11 +67,13 @@
 // autoloaders.
 $dir = dirname(__DIR__);
 require_once $dir . '/src/Xhgui/Config.php';
-Xhgui_Config::load($dir . '/config/config.default.php');
-if (file_exists($dir . '/config/config.php')) {
-    Xhgui_Config::load($dir . '/config/config.php');
+$configDir = defined('XHGUI_CONFIG_DIR') ? XHGUI_CONFIG_DIR : $dir . '/config/';
+if (file_exists($configDir . 'config.php')) {
+    Xhgui_Config::load($configDir . 'config.php');
+} else {
+    Xhgui_Config::load($configDir . 'config.default.php');
 }
-unset($dir);
+unset($dir, $configDir);
```

Comment:
If we add this config path constant to the collector package, I can get this code updated to use the collector. Its something that has been on my todo list for quite a while anyways.

Reply:
Ok. I added the changes (and some more) to https://github.com/perftools/xhgui-collector and opened a PR (https://github.com/perftools/xhgui-collector/pull/3)

Reply:
Thanks! Once that is merged I'll fix up this header file so it uses the collector but is still backwards compatible.